designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,410 @@
+warn : NU3027: The signature should be timestamped to enable long-term signature validity after the signing certificate has expired.
+
+Package signature validation failed.
+```
+
+</details>
+
+#### `Debian case - Verifying author signed package with untrusted timestamping signing certificate`
+
+<details>
+<summary>output</summary>
+
+```
+dotnet nuget verify "package.nupkg" -v n
+Verifying packageA.1.0.0
+C:\Users\kapenaga\Downloads\package.nupkg
+Signature Hash Algorithm: SHA256
+
+Signature type: Author
+Verifying the author primary signature with signing certificate:
+  Subject name: CN=test
+  SHA1 hash: B0A2B3B1695AB8361B1D2B14A9F5D64136E26380
+  SHA256 hash: 89A2B6EB529E0AEBF0D11C8A18A846C7B8D1290791B6BF494BAFEC299F2EAAB2
+  Issued by: CN=test
+  Valid from: 1/29/2021 12:28:11 PM to 02/29/2022 1:28:11 PM
+Timestamp: 2/3/2021 3:36:12 PM
+Verifying author primary signature's timestamp with timestamping service certificate:
+  Subject Name: CN=Entrust Timestamp Authority - TSA1, O="Entrust, Inc.", L=Ottawa, S=Ontario, C=CA
+  SHA1 hash: 94A59549D6E7CE926ED8D2625E1C2ADEBA04C37F
+  SHA256 hash: 950A26FDC7C02018E9F791A95C38F26EEF3DA43267CAB0CD15A555AF631072C9
+  Issued by: CN=Entrust Timestamping CA - TS1, OU="(c) 2015 Entrust, Inc. - for authorized use only", OU=See www.entrust.net/legal-terms, O="Entrust, Inc.", C=US
+  Valid from: 7/22/2020 3:33:29 PM to 12/29/2030 4:29:23 PM
+
+Finished with 1 errors and 2 warnings.
+error: NU3028: The author primary signature's timestamping certificate is not trusted by the trust provider.
+warn : NU3028: The author primary signature's timestamp found a chain building issue: The revocation function was unable to check revocation because the revocation server could not be reached. For more information, visit https://aka.ms/certificateRevocationMode.
+warn : NU3028: The author primary signature's timestamp found a chain building issue: RevocationStatusUnknown: The revocation function was unable to check revocation for the certificate.
+
+Package signature validation failed.
+```
+
+</details>
+
+#### `Debian case - Verifying expired author signed package with untrusted timestamping signing certificate`
+
+<details>
+<summary>output</summary>
+
+```
+dotnet nuget verify "AuthorExpired.1.0.0.nupkg" -v n
+Verifying packageA.1.0.0
+C:\Users\kapenaga\Downloads\package.nupkg
+Signature Hash Algorithm: SHA256
+
+Signature type: Author
+Verifying the author primary signature with signing certificate:
+  Subject name: CN=test
+  SHA1 hash: B0A2B3B1695AB8361B1D2B14A9F5D64136E26380
+  SHA256 hash: 89A2B6EB529E0AEBF0D11C8A18A846C7B8D1290791B6BF494BAFEC299F2EAAB2
+  Issued by: CN=test
+  Valid from: 1/29/2021 12:28:11 PM to 02/29/2021 1:28:11 PM
+Timestamp: 2/3/2021 3:36:12 PM
+Verifying author primary signature's timestamp with timestamping service certificate:
+  Subject name: CN=NuGet Test Root Certificate Authority (40998d55-3d73-4a3b-a689-55e30c1fac3c), O=NuGet, L=Redmond, S=WA, C=US
```

Comment:
Maybe I missed something, but why is this the root?  (Is the NuGet test timestamp server signing with the root cert?)

Reply:
`NuGet Test Root Certificate Authority` is used as root for testing purposes.

Reply:
@bartonjs - Thank you for taking the time to review this spec. Please let us know if you have any additional questions so that I can address them ASAP.